pkg/generator/test_constants.go

```diff
@@ -10,4 +10,5 @@ const (
 	appVersion     = "v1alpha1"
 	appGroupName   = "app.example.com"
 	appProjectName = "app-operator"
+	errorMessage   = "Want:\n %vGot:\n %v"
```

Comment:
By looking at the output error msg, there is an extra empty space before first line:
`" apiVersion: app.example.com/v1alpha1"`.
we might not want to have empty space before `%v`.

Suggest to have `"Want:\n%vGot:\n%v"` instead of `"Want:\n %vGot:\n %v"`